src/vs/workbench/contrib/tags/electron-browser/workspaceTagsService.ts

```diff
@@ -292,13 +293,13 @@ export class WorkspaceTagsService implements IWorkspaceTagsService {
 			"workspace.py.playwright" : { "classification": "SystemMetaData", "purpose": "FeatureInsight", "isMeasurement": true }
 		}
 	*/
-	private async resolveWorkspaceTags(): Promise<Tags> {
+	private resolveWorkspaceTags(): Promise<Tags> {
 		const tags: Tags = Object.create(null);
 
 		const state = this.contextService.getWorkbenchState();
```

Comment:
The `Uri.joinPath` utility would be good to use here

Reply:
Updated~